cloud/keystone_test.go

```diff
@@ -368,19 +373,52 @@ var _ = Describe("Keystone client", func() {
 			})
 
 			It("Should read tokens scoped to admin project", func() {
-				server.AppendHandlers(
-					ghttp.RespondWithJSONEncoded(201, getV3TokensAdminResponse()),
-					ghttp.RespondWithJSONEncoded(200, getV3TokensAdminResponse()),
-				)
+				setResponses(getV3TokensAdminResponse())
 				setupV3Client()
+
 				auth, err := client.VerifyToken(token)
+
 				Expect(err).To(BeNil())
 				Expect(auth.TenantID()).To(Equal("admin-project-id"))
 				Expect(auth.TenantName()).To(Equal("admin-project"))
 				Expect(auth.DomainID()).To(Equal("default"))
 				Expect(auth.DomainName()).To(Equal("default"))
 				Expect(auth.IsAdmin()).To(BeTrue())
 			})
+
+			It("Should reject admin token with multiple roles", func() {
+				multipleRoles := []interface{}{
+					map[string]interface{}{
+						"id":   "51cc68287d524c759f47c811e6463340",
```

Comment:
I've seen those roles in `keystone_test_utils.go`.
Can they be extracted to a function, for example:
```go
func getMemberRole() map[string]interface{} {
    return map[string]interface{
        "id": "51cc68287d524c759f47c811e6463340",
        "name": "Member",
    }
}
func getAdminRole() map[string]interface{} {
    return map[string]interface{
        "id": "7f0ea059b6d84029b60c18169d3c1d9a",
        "name": "admin",
    }
}
```
This would also be helpful to give context about the ids, since currently it's tricky to tell why they have such specific values.

Reply:
I actually taken then from there but didn't really finish cleaning up. Done.

Regarding their IDs: I have no idea, they were added way before. I suspect they were simply copied from some Keystone installation to serve as examples.